api/src/test/java/com/epam/pipeline/manager/pipeline/PipelineRunManagerLaunchTest.java

```diff
@@ -328,6 +356,28 @@ public void workerRunShouldUseParentRunCloudRegionWithParentRunIdPassedAsParamet
         assertThat(pipelineRun.getInstance().getCloudRegionId(), is(NON_DEFAULT_REGION_ID));
     }
 
+    @Test
+    public void shouldLoadRunsActivityStats() {
+        final PipelineRun pipelineRun = getPipelineRun(ID, TEST_USER);
+        final PipelineRun pipelineRun2 = getPipelineRun(ID_2, TEST_USER);
+        doReturn(pipelineRun).when(pipelineRunDao).loadPipelineRun(anyLong());
+        doReturn(newArrayList(pipelineRun, pipelineRun2)).when(pipelineRunDao)
+                .loadPipelineRunsActiveInPeriod(eq(TEST_PERIOD), eq(TEST_PERIOD.plusHours(HOURS_18)));
+        doReturn(getStatusMap()).when(runStatusManager).loadRunStatus(anyListOf(Long.class));
+
+        launchPipelineRun(TEST_PERIOD, TEST_PERIOD.plusHours(HOURS_12));
+        launchPipelineRun(TEST_PERIOD.plusHours(HOURS_18), null);
```

Comment:
As long as we write a unit test that tests a single `pipelineRunManager.loadRunsActivityStats` method please mock and verify execution of a single method only. No need to launch any pipelines within fully mocked environment. As a result the test should be significantly simplified.

Reply:
Done!